### ISSUE TITLE
Pre-train with Adam optimizer

### ISSUE THREAD

**Issue by yichen928** [facebookresearch/DepthContrast#27]
Hello, thank you for this great work.

I find you apply SGD+momentum optimizer to the pre-training, but other optimizers are used for fine-tuning. I would like to know that have you tried other optimizers such as Adam, AdamW and LARS for pre-training? And will other choices lead to worse performance of pre-training?

Thank you very much.

**Comment by zaiweizhang** [comment]
No. I have not tried Adam, AdamW and LARS. We choose SGD + momentum because MoCo is using that and we use MoCo loss formulation for our framework.
